.circleci/config.yml

```diff
@@ -13,6 +13,12 @@ executors:
     resource_class: medium
     environment:
       NODE_OPTIONS: --max_old_space_size=3072
+  node-linux-medium:
+    machine:
+      image: ubuntu-2404:2024.05.1
+    resource_class: medium #// linux medium: 2 CPUs, 7.5 GB RAM, 10 credits/min
+    environment:
+      NODE_OPTIONS: --max_old_space_size=6144
```

Comment:
Curious that we're allowing for a larger heap here than on `medium+`. I guess there should be adequate room though, as long as we're not running multiple Node.js processes in one job.